src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        Figure figure = new Circle();

```

Comment:
we don't need to create 'Circle'. can we return the object right from the switch-case?